lib/core/src/lib/datatable/components/datatable/datatable.component.ts

```diff
@@ -285,6 +286,9 @@ export class DataTableComponent implements OnInit, AfterContentInit, OnChanges,
 
     ngOnChanges(changes: SimpleChanges) {
         this.initAndSubscribeClickStream();
+        if(this.selectedRowId) {
```

Comment:
missing unit tests for all the added code

Reply:
working on it.

Reply:
unit test added for the change.